content/hi/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,649 @@
+ ---
+reviewers:
+- caesarxuchao
+- dchen1107
+title: नोड्स
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+Kubernetes _Nodes_ पर चलने के लिए पॉड्स में कंटेनर रखकर आपका {{< glosary_tooltip text="workload" term_id="workload" >}} चलाता है।
+क्लस्टर के आधार पर एक नोड वर्चुअल या भौतिक मशीन हो सकता है। प्रत्येक नोड
+द्वारा प्रबंधित किया जाता है
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}}
+और चलाने के लिए आवश्यक सेवाएं शामिल हैं
+{{< glosary_tooltip text="Pods" term_id="pod" >}}.
```

Comment:
Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

```suggestion
{{< glossary_tooltip text="पॉड्स" term_id="pod" >}}.
```